case_study1_airline_safety.Rmd

```diff
@@ -0,0 +1,106 @@
+# Case study #1 : Airline Safety data
+
+    ```{r setup, include=FALSE}
+    knitr::opts_chunk$set(include = FALSE)
+    library(ggplot2)
+    library(ggpubr)
+    library(haven)
+    library(dplyr)
+    ```
+
+## Setup
+
+This case study is based on an [article](https://fivethirtyeight.com/features/should-travelers-avoid-flying-airlines-that-have-had-crashes-in-the-past/) from [FiveThirtyEight](https://fivethirtyeight.com/) published by [Nate Silver](https://fivethirtyeight.com/contributors/nate-silver/). Using [data](https://github.com/fivethirtyeight/data/tree/master/airline-safety) from the [Aviation Safety Network’s database](https://aviation-safety.net/index.php), we want to see if past safety records can predict future risk of accidents. In other words "Should Travelers Avoid Flying Airlines That Have Had Crashes in the Past?"
+
+    ```{r include = FALSE}
+    ## need to find the best way to let students download the data 
+    data=read.csv2("~/Documents/TA_Econometrics/Case_studies/airline_safety/data-master/airline-safety/airline-safety_long.csv", sep = ";")
+    ```
```

Comment:
is the link correct? i cannot see any file called `_long.csv`? also, i don't hve the variable `type` in the downloaded data?

Reply:
no it was a mistake : I'm the only one to have the dataset at this point because I changed the data that you probably downloaded, sorry

Reply:
if it's important we could distribute your version of the data?

Reply:
yes, actually it is just a reshape (wide to long) of the original dataset (https://github.com/fivethirtyeight/data/tree/master/airline-safety). Otherwise there was nothing to group by etc ...